src/components/experiences/social-login/index.js

```diff
@@ -26,30 +28,27 @@ function SocialLogin(props) {
         locale,
     } = props;
     return (
-        <article className="rainbow-auth-firebase-social-login_buttons-container">
+        <StyledArticle>
             <div>
-                <Button
-                    className="rainbow-auth-firebase-social-login_button rainbow-auth-firebase-social-login_facebook-button"
+                <StyledFacebookButton
                     onClick={loginWithFacebook}
                     isLoading={isLoadingFacebook}>
-                    <FacebookIcon className="rainbow-auth-firebase-social-login_social-icon" />
+                    <StyledIcon as={FacebookIcon} />
```

Comment:
you don't need pass `as={FacebookIcon}` if you pass it when you define the `styled-component`